### ISSUE TITLE
a lot of things missing

### ISSUE THREAD

**Issue by fang798** [Bala93/CRac#1]
In CRac/calibrate/engine/lagrangian_med_trainer.py, there seem to be a lot of things missing, such as:
from calibrate.net import ModelWithTemperature
from calibrate.losses import LogitMarginL1
from calibrate.utils.bw_const_helper import transform_batch, get_lambda_maps
I don't know if it's a problem with my operation

**Comment by Bala93** [comment]
I guess, these ones are not used for the experiments. Can you comment and check if you are able to run the codebase ?

**Comment by fang798** [comment]
> I guess, these ones are not used for the experiments. Can you comment and check if you are able to run the codebase ?

For example, when running tools/train_net.py, it will :
"import Trainer, MedSegmentTrainer, LagrangianMedSegmentTrainer from calibrate.engine", 
and in calibrate/engine/trainer.py it will :
"import ModelWithTemperature from calibrate.net, and LogitMarginL1 from calibrate.losses". 
But these do not exist.

**Comment by fang798** [comment]
> I guess, these ones are not used for the experiments. Can you comment and check if you are able to run the codebase ?

For example, when running tools/train_net.py, it will :
from calibrate.engine import Trainer, MedSegmentTrainer, LagrangianMedSegmentTrainer
and then in calibrate/engine/trainer.py it will :
from calibrate.net import ModelWithTemperature
from calibrate.losses import LogitMarginL1
But these do not exist.

**Comment by Bala93** [comment]
We don't need these two to test CRaC, from calibrate.net import ModelWithTemperature
from calibrate.losses import LogitMarginL1 -- Can you remove them and give this one a try ?

**Comment by Bala93** [comment]
For more loss functions, I would refer you to https://github.com/Bala93/MarginLoss/tree/master
